### PR TITLE
DOCS-47 - Add hidden instructions for Cloud SIEM (Kanso)

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request adds hidden Kanso instructions to Cloud SIEM articles.

**IMPORTANT**: For the purposes of review, in this PR the Kanso instructions are shown. Once review of this PR is complete, I'll add comments to hide the Kanso instructions before publishing the PR.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[DOCS-47](https://sumologic.atlassian.net/browse/DOCS-47)

### CODE REVIEW

File: docs/cse/administration/mitre-coverage.md [hunk 1]
```diff
@@ -37,7 +37,9 @@ Watch this micro lesson to learn about the MITRE ATT&CK Threat Coverage Explorer
 
 ## MITRE ATT&CK Threat Coverage Explorer
 
-To open the MITRE ATT&CK Threat Coverage Explorer, select **Content** in the top menu and select **MITRE ATT&CK Coverage**.
+[**Classic UI**](/docs/cse/introduction-to-cloud-siem/#classic-ui). To open the MITRE ATT&CK Threat Coverage Explorer, in the top menu select **Content > MITRE ATT&CK Coverage**.
```
Comment (JV0812) [inline docs/cse/administration/mitre-coverage.md:40]:
```suggestion
[**Classic UI**](/docs/cse/introduction-to-cloud-siem/#classic-ui). To open the MITRE ATT&CK Threat Coverage Explorer, in the top menu select **Content > MITRE ATT&CK Coverage**.<br/>[**New UI**](/docs/cse/introduction-to-cloud-siem/#new-ui). To open the MITRE ATT&CK Threat Coverage Explorer, in the main Sumo Logic menu select **Cloud SIEM > MITRE ATT&CK Coverage**. You can also click the **Go To...** menu at the top of the screen and select **MITRE ATT&CK Coverage**. 
```

File: docs/cse/administration/mitre-coverage.md [hunk 2]
```diff
@@ -37,7 +37,9 @@ Watch this micro lesson to learn about the MITRE ATT&CK Threat Coverage Explorer
...
+
```
Comment (JV0812) [inline docs/cse/administration/mitre-coverage.md:41]:
```suggestion
```

File: docs/cse/administration/mitre-coverage.md [hunk 3]
```diff
@@ -37,7 +37,9 @@ Watch this micro lesson to learn about the MITRE ATT&CK Threat Coverage Explorer
...
+[**New UI**](/docs/cse/introduction-to-cloud-siem/#new-ui). To open the MITRE ATT&CK Threat Coverage Explorer, in the main Sumo Logic menu select **Cloud SIEM > MITRE ATT&CK Coverage**. You can also click the **Go To...** menu at the top of the screen and select **MITRE ATT&CK Coverage**.
```
Comment (JV0812) [inline docs/cse/administration/mitre-coverage.md:42]:
```suggestion
```

File: docs/cse/get-started-with-cloud-siem/about-cse-insight-ui.md [hunk 4]
```diff
@@ -11,7 +11,9 @@ This topic describes the Cloud SIEM UI for working with Insights. 
 
 ## Insights list page
 
-To open the **Insights** page, click the icon to the right of the search bar at the top of the Cloud SIEM UI.
+[**Classic UI**](/docs/cse/introduction-to-cloud-siem/#classic-ui). To open the Insights page, click **Insights** at the top of the screen.
```
Comment (JV0812) [inline docs/cse/get-started-with-cloud-siem/about-cse-insight-ui.md:14]:
```suggestion
[**Classic UI**](/docs/cse/introduction-to-cloud-siem/#classic-ui). To open the Insights page, click **Insights** at the top of the screen. <br/>[**New UI**](/docs/cse/introduction-to-cloud-siem/#new-ui). To open the Insights page, in the main Sumo Logic menu select **Cloud SIEM > Insights**. You can also click **Go To...** at the top of the screen and select **Insights**.
```

File: docs/cse/get-started-with-cloud-siem/about-cse-insight-ui.md [hunk 5]
```diff
@@ -11,7 +11,9 @@ This topic describes the Cloud SIEM UI for working with Insights. 
...
+
```
Comment (JV0812) [inline docs/cse/get-started-with-cloud-siem/about-cse-insight-ui.md:15]:
```suggestion
```

File: docs/cse/get-started-with-cloud-siem/about-cse-insight-ui.md [hunk 6]
```diff
@@ -11,7 +11,9 @@ This topic describes the Cloud SIEM UI for working with Insights. 
...
+[**New UI**](/docs/cse/introduction-to-cloud-siem/#new-ui). To open the Insights page, in the main Sumo Logic menu select **Cloud SIEM > Insights**. You can also click **Go To...** at the top of the screen and select **Insights**.
```
Comment (JV0812) [inline docs/cse/get-started-with-cloud-siem/about-cse-insight-ui.md:16]:
```suggestion
```

File: docs/cse/records-signals-entities-insights/view-manage-entities.md [hunk 7]
```diff
@@ -54,7 +54,9 @@ When a Signal is fired, if an Entity doesn’t already exist in Cloud SIEM for t
 
 ## About the Entities list page
 
-To view the **Entities** page, click **Entities** in the top menu of the Cloud SIEM UI.
+[**Classic UI**](/docs/cse/introduction-to-cloud-siem/#classic-ui). To view Entities, click **Entities** at the top of the screen.
+
```
Comment (JV0812) [inline docs/cse/records-signals-entities-insights/view-manage-entities.md:58]:
```suggestion
```

File: docs/cse/records-signals-entities-insights/view-manage-entities.md [hunk 8]
```diff
@@ -54,7 +54,9 @@ When a Signal is fired, if an Entity doesn’t already exist in Cloud SIEM for t
 
 ## About the Entities list page
 
-To view the **Entities** page, click **Entities** in the top menu of the Cloud SIEM UI.
+[**Classic UI**](/docs/cse/introduction-to-cloud-siem/#classic-ui). To view Entities, click **Entities** at the top of the screen.
```
Comment (JV0812) [inline docs/cse/records-signals-entities-insights/view-manage-entities.md:57]:
```suggestion
[**Classic UI**](/docs/cse/introduction-to-cloud-siem/#classic-ui). To view Entities, click **Entities** at the top of the screen. <br/>[**New UI**](/docs/cse/introduction-to-cloud-siem/#new-ui). To view Entities, in the main Sumo Logic menu select **Cloud SIEM > Entities**. You can also click the **Go To...** menu at the top of the screen and select **Entities**.
```

File: docs/cse/records-signals-entities-insights/view-manage-entities.md [hunk 9]
```diff
@@ -54,7 +54,9 @@ When a Signal is fired, if an Entity doesn’t already exist in Cloud SIEM for t
...
+
+[**New UI**](/docs/cse/introduction-to-cloud-siem/#new-ui). To view Entities, in the main Sumo Logic menu select **Cloud SIEM > Entities**. You can also click the **Go To...** menu at the top of the screen and select **Entities**.
```
Comment (JV0812) [inline docs/cse/records-signals-entities-insights/view-manage-entities.md:59]:
```suggestion
```

File: docs/cse/rules/about-cse-rules.md [hunk 10]
```diff
@@ -9,7 +9,13 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 A Cloud SIEM rule is logic that fires based on information in incoming Records. When a rule fires, it creates a Signal.
 
+[**Classic UI**](/docs/cse/introduction-to-cloud-siem/#classic-ui). To view Rules, in the top menu select **Content > Rules**.
+
```
Comment (JV0812) [inline docs/cse/rules/about-cse-rules.md:13]:
```suggestion
```

File: docs/cse/rules/about-cse-rules.md [hunk 11]
```diff
@@ -9,7 +9,13 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 A Cloud SIEM rule is logic that fires based on information in incoming Records. When a rule fires, it creates a Signal.
 
+[**Classic UI**](/docs/cse/introduction-to-cloud-siem/#classic-ui). To view Rules, in the top menu select **Content > Rules**.
```
Comment (JV0812) [inline docs/cse/rules/about-cse-rules.md:12]:
```suggestion
[**Classic UI**](/docs/cse/introduction-to-cloud-siem/#classic-ui). To view Rules, in the top menu select **Content > Rules**.  <br/>[**New UI**](/docs/cse/introduction-to-cloud-siem/#new-ui). To view Rules, in the main Sumo Logic menu select **Cloud SIEM > Rules**. You can also click the **Go To...** menu at the top of the screen and select **Rules**. 
```

File: docs/cse/rules/about-cse-rules.md [hunk 12]
```diff
@@ -9,7 +9,13 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+
+[**New UI**](/docs/cse/introduction-to-cloud-siem/#new-ui). To view Rules, in the main Sumo Logic menu select **Cloud SIEM > Rules**. You can also click the **Go To...** menu at the top of the screen and select **Rules**.
```
Comment (JV0812) [inline docs/cse/rules/about-cse-rules.md:14]:
```suggestion
```

File: docs/cse/records-signals-entities-insights/view-records-signal.md [hunk 13]
```diff
@@ -6,9 +6,14 @@ description: Learn how to view records associated with a signal.
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-Cloud SIEM uses rules to evaluate incoming records, and when the conditions of a rule are met, generates a signal. This topic explains how to view records associated with a signal in Cloud SIEM. 
+Cloud SIEM uses rules to evaluate incoming records, and when the conditions of a rule are met, generates a signal. This topic explains how to view records associated with a signal in Cloud SIEM.
+ 
 ## View record details
 
+[**Classic UI**](/docs/cse/introduction-to-cloud-siem/#classic-ui). To view Signals, click **Signals** at the top of the screen.
```
Comment (JV0812) [inline docs/cse/records-signals-entities-insights/view-records-signal.md:13]:
```suggestion
[**Classic UI**](/docs/cse/introduction-to-cloud-siem/#classic-ui). To view Signals, click **Signals** at the top of the screen. <br/>[**New UI**](/docs/cse/introduction-to-cloud-siem/#new-ui). To view Signals, in the main Sumo Logic menu select **Cloud SIEM > Signals**. You can also click the **Go To...** menu at the top of the screen and select **Signals**.
```

File: docs/cse/records-signals-entities-insights/view-records-signal.md [hunk 14]
```diff
@@ -6,9 +6,14 @@ description: Learn how to view records associated with a signal.
...
+
```
Comment (JV0812) [inline docs/cse/records-signals-entities-insights/view-records-signal.md:14]:
```suggestion
```

File: docs/cse/records-signals-entities-insights/view-records-signal.md [hunk 15]
```diff
@@ -6,9 +6,14 @@ description: Learn how to view records associated with a signal.
...
+[**New UI**](/docs/cse/introduction-to-cloud-siem/#new-ui). To view Signals, in the main Sumo Logic menu select **Cloud SIEM > Signals**. You can also click the **Go To...** menu at the top of the screen and select **Signals**.
```
Comment (JV0812) [inline docs/cse/records-signals-entities-insights/view-records-signal.md:15]:
```suggestion
```

File: docs/cse/get-started-with-cloud-siem/cse-heads-up-display.md [hunk 16]
```diff
@@ -8,6 +8,10 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 This topic describes Cloud SIEM *Heads Up Display (HUD)*, the landing page for the Cloud SIEM UI. The HUD provides an at-a-glance overview of Insight status and activity.
 
+[**Classic UI**](/docs/cse/introduction-to-cloud-siem/#classic-ui). To access the HUD, in the main Sumo Logic menu select **Cloud SIEM**.
```
Comment (JV0812) [inline docs/cse/get-started-with-cloud-siem/cse-heads-up-display.md:11]:
```suggestion
[**Classic UI**](/docs/cse/introduction-to-cloud-siem/#classic-ui). To access the HUD, in the main Sumo Logic menu select **Cloud SIEM**. <br/>[**New UI**](/docs/cse/introduction-to-cloud-siem/#new-ui). To access the HUD, in the main Sumo Logic menu select **Cloud SIEM > SIEM Overview**. You can also click **Go To...** at the top of the screen and select **SIEM Overview**.
```

File: docs/cse/get-started-with-cloud-siem/cse-heads-up-display.md [hunk 17]
```diff
@@ -8,6 +8,10 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+
```
Comment (JV0812) [inline docs/cse/get-started-with-cloud-siem/cse-heads-up-display.md:12]:
```suggestion
```

**Review by JV0812 — APPROVED** [review]
Great work @jpipkin1.  LGTM, some mirror comments just to maintain the similarity.

**Comment by jpipkin1** [comment]
Thanks for the review, @JV0812!

File: docs/cse/administration/mitre-coverage.md (hunk 1 above)
Comment (jpipkin1) [reply docs/cse/administration/mitre-coverage.md:40]: @JV0812, I decided to go back to placing a space between all the sets of instructions you wanted to place together.

**Pull request merged**